### ISSUE TITLE
Low accuracy results in the meta-knowledge for classification tasks??

### ISSUE THREAD

**Issue by jsebanaz90** [automl/auto-sklearn#695]
Hello,

I'm exploring the meta-base of knowledge for classification tasks. More precisely, I'm focused on the metadata within the algorithms runs of "accuracy_binary.classification_dense".

According to the autosklearn paper, this meta-knowledge is estimated in an off-line phase using Bayesian optimisation; however, exploring the algorithms runs, there are accuracy results that are quite low.

For instance, in the case of "246,1.0,74,0.010606060606060619,ok", for dataset "246" the best accuracy obtained is "0.01..", and for the dataset "244" there is a similar value.

I would like to know if the accuracies are normalised or if any preprocessing process for them was carried out. How could I interpret these values?

Thanks!

**Comment by mfeurer** [comment]
> Does the attribute "instance_id STRING" correspond to the id of each dataset? i found that the datasets ids in the supplementary file of the 2015 paper do not match with these "instance_id STRINGs".

The IDs are IDs of OpenML tasks and each task references a dataset itself. You can retrieve them by replacing the task id in https://www.openml.org/t/2120 with the respective one. However, the datasets are indeed slightly different from the ones we used in 2015.

> On the other hand, looking at the attribute "accuracy NUMERIC" which should correspond to the performance of the best ML pipelines found for every dataset, I found that there are plenty of low values. If the accuracy metric ranges from 0 to 1  being the latter high accuracy performance, why are there low values in most of the cases (accuracy values between 0.0... and 0.3...) if the Bayesian optimisation guarantees to find high-performance ML pipelines? For example:  "246,1.0,74,0.010606060606060619,ok"  means a accuracy metric of 0.1 for Ml pipeline "74" in dataset "246"?

The usage of loss and metric is somewhat inconsistent in Auto-sklearn. The number you're seeing here is actually the loss obtained by running vanilla Auto-sklearn for the respective task. We store the loss because SMAC internally minimizes the function of interest. In practice, this number is not used because there's only one entry for each task.
